apps/site/assets/js/tabbed-nav.js

```diff
@@ -0,0 +1,26 @@
+const tabbedNavSetup = () => {
+  const navTabs = document.querySelectorAll(".m-tabbed-nav__item");
+  const contentTabs = document.querySelectorAll(".m-tabbed-nav__content-item");
+
+  navTabs.forEach(item => {
+    const { tabType } = item.dataset;
+
+    item.addEventListener("click", e => {
+      e.preventDefault();
+
+      navTabs.forEach(tab => tab.classList.remove("active"));
```

Comment:
Might `classList.toggle` (in particular with [example using a conditional](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples)) be useful here?